models/studentsModel.js

```diff
@@ -0,0 +1,142 @@
+const fs = require("fs")
+const pool = require("../connection/config")
+const Convert = require("../convertTime")
+
+
+class studentsModels {
+    static showStudents(callback) {
+
+        const query = `
+            SELECT * FROM students
+            ORDER BY id ASC
+        `
+
+        pool
+        .query(query)
+        .then(res => {
+            for (let i = 0; i < res.rows.length; i++) {
+                res.rows[i].birth_date = Convert.toIndo(res.rows[i].birth_date.toString().slice(4, 15))
+                // console.log(res.rows[i])
+            }
+            callback(null, res.rows)
+        })
+        .catch(err => callback(err, null))
+    }
+
+    static findStudentByEmail(email, callback) {
+        const query = `
+            SELECT * FROM students
+            WHERE email = $1
+        `
+        // console.log(email)
+        let params = [email]
+        pool
+        .query(query, params)
+        .then(res => {
+            // console.log(res)
+            res.rows[0].birth_date = Convert.toIndo(res.rows[0].birth_date.toString().slice(4, 15))
+            callback(null, res.rows)
+        })
+        .catch(err => callback(err, null))
+    }
+
+    static postAddStudentsData(req, callback) {
+        // console.log(req)
+        const query = `
+                INSERT INTO students (first_name,last_name,email,gender,birth_date) VALUES ($1,$2,$3,$4,$5)
+        `
+        const error = this.validation(req)
+
+        if (error.length > 0) {
+            callback(error, null)
+        } else {
+            let params = [req.first_name, req.last_name, req.email, req.gender, req.birthdate]
+            pool
+            .query(query, param)
+            .then(res => callback(null, results.rows))
```

Comment:
hati-hati sama penulisan variable
```suggestion
            .then(res => callback(null, res.rows))
```